src/ui/assets/MediaSourcePanel.js

```diff
@@ -26,7 +26,7 @@ export default function MediaSourcePanel({ editor, source, searchPlaceholder, in
 
       editor.spawnGrabbedObject(node);
     },
-    [editor]
+    [editor, source.transformPivot]
```

Comment:
Thanks for catching this! In the future, please submit fixes like this as a separate PR.